src/initialize/src/controllers/tarController.ts

```diff
@@ -0,0 +1,42 @@
+/*******************************************************************************
+ * Copyright (c) 2019 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v20.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+
+import fs from 'fs-extra';
+import { exec } from 'child_process';
+import request from 'request';
+
+export function importProjectFromTar(tarFile: string, name: string, dest: string): Promise<void> {
+
+    const fileName = name + '.tar.gz';
+    const file = fs.createWriteStream(fileName);
```

Comment:
This should probably be an absolute filename, otherwise it will just be created in the current working directory (and I'm not sure exactly where that will be in all cases).

Reply:
Please use `/tmp` not `/app`. `/app` exists in the initialiser pod but the initialiser code is run directly as an npm when we are in che and `/app` doesn't exist there.
(Also /tmp is a better location for temporary files.)